### DISCUSSION TITLE
How does these nice custom folder icons created?

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1358] [General]
What software do you use to make blending in icons on the default icon folder, that looks like they've been made by apple? example:
<img width="91" alt="Без названия" src="https://user-images.githubusercontent.com/47495939/152457835-4c6e25b1-6d93-4932-aab7-28cfbe19299c.png">

**Comment by bunkerdesign** [comment]
Try using overlay blending mode in any image editor
